docs/sources/shared/flow/reference/components/rule-block-logs.md

```diff
@@ -10,36 +10,36 @@ description: Shared content, rule block logs
 headless: true
 ---
 
-The `rule` block contains the definition of any relabeling rules that can be
-applied to an input log. If more than one `rule` block is defined, the
-transformations are applied in top-down order.
+The `rule` block contains the definition of any relabeling rules that can be applied to an input log.
+The transformations are applied in top-down order if more than one `rule` block is defined.
 
-The following arguments can be used to configure a `rule`. All arguments are
-optional. Omitted fields take their default values.
+You can use the following arguments to configure a `rule`.
+All arguments are optional. Omitted fields take their default values.
 
-Name | Type | Description | Default | Required
----- | ---- | ----------- | ------- | --------
-`source_labels` | `list(string)` | The list of labels whose values are to be selected. Their content is concatenated using the `separator` and matched against `regex`. | | no
-`separator`     | `string`       | The separator used to concatenate the values present in `source_labels`. | ; | no
+Name            | Type           | Description                                                                                                                          | Default | Required
+----------------|----------------|--------------------------------------------------------------------------------------------------------------------------------------|---------|---------
+`action`        | `string`       | The relabeling action to perform.                                                                                                    | replace | no
+`modulus`       | `uint`         | A positive integer used to calculate the modulus of the hashed source label values.                                                  |         | no
 `regex`         | `string`       | A valid RE2 expression with support for parenthesized capture groups. Used to match the extracted value from the combination of the `source_label` and `separator` fields or filter labels during the `labelkeep/labeldrop/labelmap` actions. | `(.*)` | no
-`modulus`       | `uint`         | A positive integer used to calculate the modulus of the hashed source label values. | | no
-`target_label`  | `string`       | Label to which the resulting value will be written to. | | no
-`replacement`   | `string`       | The value against which a regex replace is performed, if the regex matches the extracted value. Supports previously captured groups. | $1 | no
-`action`        | `string`       | The relabeling action to perform. | replace | no
+`replacement`   | `string`       | The value against which a regular expression replace is performed if the regular expression matches the extracted value. Supports previously captured groups. | $1      | no
```

Comment:
Just noticed that the default value is missing code block wrapping (and double quotes to denote it as a string value):

```suggestion
`replacement`   | `string`       | The value against which a regular expression replace is performed if the regular expression matches the extracted value. Supports previously captured groups. | `"$1"`      | no
```